File: executable/r/multiple-axes-double.r
library(plotly)
p <- plotly(username='test-runner', key='9h29fe3l0x')

trace1 <- list(
  x = c(1, 2, 3), 
  y = c(40, 50, 60), 
  name = "yaxis data"
)
trace2 <- list(
  x = c(2, 3, 4), 
  y = c(4, 5, 6), 
  yaxis = "y2", 
  name = "yaxis2 data"
)

layout <- list(
  title = "Double Y Axis Example", 
  yaxis = list(title = "yaxis title"), 
  yaxis2 = list(
    title = "yaxis2 title", 
    titlefont = list(color = "rgb(148, 103, 189)"), 
    tickfont = list(color = "rgb(148, 103, 189)"), 
    overlaying = "y", 
    side = "right"
  )
)

response <- p$plotly(trace0, trace1, kwargs=list(layout=layout, filename="multiple-axes-double", fileopt="overwrite"))
url <- response$url
filename <- response$filename

File: executable/python/multiple-axes-double.py
import plotly.plotly as py
from plotly.graph_objs import *

py.sign_in('test-runner', '9h29fe3l0x')

trace1 = Scatter(
    x=[1, 2, 3],
    y=[40, 50, 60],
    name='yaxis data'
)
trace2 = Scatter(
    x=[2, 3, 4],
    y=[4, 5, 6],
    name='yaxis2 data',
    yaxis='y2'
)
data = Data([trace1, trace2])
layout = Layout(
    title='Double Y Axis Example',
    yaxis=YAxis(
        title='yaxis title'
    ),
    yaxis2=YAxis(
        title='yaxis2 title',
        titlefont=Font(
            color='rgb(148, 103, 189)'
        ),
        tickfont=Font(
            color='rgb(148, 103, 189)'
        ),
        side='right',
        overlaying='y'
    )
)
fig = Figure(data=data, layout=layout)

plot_url = py.plot(fig, filename='multiple-axes-double')